src/cryptoadvance/specter/server.py

```diff
@@ -62,6 +62,14 @@ def calc_module_name(config):
         return "cryptoadvance.specter.config." + config
 
 
+def flash(*args, **kwargs):
```

Comment:
I am unsure why this double-definition of flash should ever work and not create a recursive loop....

Reply:
Why move the flash function in server and not in util?

Reply:
Good point. Renamed it. I think everything which is called "util" or "helper" or something like that is suppose to not be dependent on the state of objects.
Doing something like `current_app.specter.some_manager.some_method()` in a util package is imho a very bad idea. There, you want to have deterministic functions or objects which don't do much magic apart maybe from using external libraries which themself also are deterministic.

So `server.py` was probably also a bad idea and i moved it to `server_endpoints/__init__.py`